troupon/static/scss/partials/_sections.scss

```diff
@@ -397,3 +397,28 @@ footer {
     margin-top: 30px;
   }
 }
+
+// messages
+.strip {
+  padding: 10px 5px 10px;

```

Comment:
Shorthand form for property `padding` should be written more concisely as `10px 5px` instead of `10px 5px 10px`